test/unit/module/core/scan_test.hpp

```diff
@@ -0,0 +1,86 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
```

Comment:
neither first true nor count are a san. They might be reductions but this can only be any bool test too